Utils/src/SignalScanProducer.cc

```diff
@@ -202,6 +242,26 @@ void SignalScanProducer::getSUSYModelInfo(std::string comment){
 	signalParameters_.push_back(lspMass_);
 }
 
+//parse model comment for SUSYGenPart
+void SignalScanProducer::getSUSYGenPartModelInfo(std::string comment){
```

Comment:
just make this take `const GenLumiInfoHeader& gen` to avoid messing with `getSUSYComment()`

Reply:
Changed